src/passes/OptimizeInstructions.cpp

```diff
@@ -1570,6 +1570,26 @@ struct OptimizeInstructions
         return InvalidBinary;
     }
   }
+
+  bool isSpecialSymmetric(Binary* binary) {
+    switch (binary->op) {
```

Comment:
This way we can consolidate the logic more and avoid having to come up with a new function name:

```suggestion
  bool isSymmetric(Binary* binary) {
    if (Properties::isSymmetric(binary)) {
      return true;
    }
    switch (binary->op) {
```